nntrainer/src/conv2d_layer.cpp

```diff
@@ -315,20 +312,32 @@ sharedConstTensors Conv2DLayer::backwarding(sharedConstTensors derivatives,
    *       x [(input_dim.height() + padding[0]*2)
    *           *(input_dim.width() + padding[1]*2) (width)]
    */
+  using uint = unsigned int;
+
+  Tensor ret(in_dim.batch(), in_dim.channel(), in_dim.height() + padding[0] * 2,
+             in_dim.width() + padding[1] * 2);
+  ret.setZero();
```

Comment:
most likely out of scope for this PR
but is this serZero needed? ret is used with conv2d_gemm and gemm does not need zeroing out output if beta (multiplier for output) is set to zero which is done at 545.
If removing this line changes output, then maybe another bug exist?

Reply:
I think it is not needed. I'll remove this. If an error exists I'll leave an issue instead.

Reply:
Erased